ci/bin/concourse_job_performance.py

```diff
@@ -282,17 +287,24 @@ def get_builds_to_examine(builds, build_count):
     succeeded, failed, aborted, errored, pending, started = sieve(builds, lambda b: b['status'], *statuses)
     completed_builds = succeeded + failed
     completed_builds.sort(key=itemgetter('id'), reverse=True)
-    builds_to_analyze = completed_builds[:build_count] if build_count else completed_builds
+    if build_count == 0:
+        count_of_builds_to_return = len(completed_builds)
+    else:
+        count_of_builds_to_return = build_count
+
+    builds_to_analyze = completed_builds[:count_of_builds_to_return] if count_of_builds_to_return else completed_builds
     logging.debug(f"{len(aborted)} aborted builds in examination range: {list_and_sort_by_name(aborted)}")
     logging.debug(f"{len(errored)} errored builds in examination range: {list_and_sort_by_name(errored)}")
     logging.debug(f"{len(pending)} pending builds in examination range: {list_and_sort_by_name(pending)}")
     logging.debug(f"{len(started)} started builds in examination range: {list_and_sort_by_name(started)}")
+    del builds[0:(len(builds)-len(completed_builds))]
```

Comment:
I am worried here if we have a run set that has a higher-indexed run that finishes before a lower-indexed run.  I'm not entirely sure on how the `curl` request would return in that case, either.

It's an edge-case to be sure, but this might be more stable as a filtered list comprehension.

```
builds = [job for job in builds if job not in succeeded + failed]
```